examples/private_set_intersection/README.md

```diff
@@ -0,0 +1,16 @@
+# Private Set Intersection
```

Comment:
Should we move these instructions to `examples/README.md` so they could apply to all examples?

Reply:
I think it needs more work to make them applicable to the other examples (do any other examples actually rely on signatures and uploading modules?), perhaps let's do it in a separate PR.

Reply:
Agree
Currently we only have Aggregator and Private Set Intersection that use remotely downloaded modules